### PR TITLE
Move searchsorted functions out of helpdb

### PR DESCRIPTION
**Pull request by kshyatt**
I built the docs locally and they rendered correctly :D

### CODE REVIEW

File: base/sort.jl [hunk 1]
```diff
@@ -219,6 +219,72 @@ for s in [:searchsortedfirst, :searchsortedlast, :searchsorted]
     end
 end
 
+@doc """
```
Comment (tkelman) [inline base/sort.jl]: does this need the doc macro?

Comment (kshyatt) [inline base/sort.jl]: I think so - I followed the example [here](https://docs.julialang.org/en/latest/manual/documentation/#Advanced-Usage-1).

**Comment by kshyatt** [comment]
OK to go?

File: base/sort.jl (hunk 1 above)
Comment (tkelman) [inline base/sort.jl]: did it work without? didn't seem to need it in helpdb

File: base/sort.jl [hunk 2]
```diff
@@ -219,6 +219,72 @@ for s in [:searchsortedfirst, :searchsortedlast, :searchsorted]
...
+    searchsorted(a, x, [by=<transform>,] [lt=<comparison>,] [rev=false])
+
+Returns the range of indices of `a` which compare as equal to `x` (using binary search)
```
Comment (tkelman) [inline base/sort.jl]: imperative ?

**Pull request merged**